EIPS/eip-695.md

```diff
@@ -20,7 +20,7 @@ Include `eth_chainId` method in `eth_`-namespaced JSON-RPC methods.
 The `eth_chainId` method should return a single STRING result
 for an integer value in hexadecimal format, describing the
 currently configured `CHAIN_ID` value used for signing replay-protected transactions,
-introduced via [EIP-155](https://eips.ethereum.org/EIPS/eip-155).
+introduced by [EIP 155](https://eips.ethereum.org/EIPS/eip-155).
```

Comment:
To be fair EIP-155 (with the dash) is the recommended notation IIRC.

Reply:
Hm, [EIP 1](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md) is contradictory on the matter. It has `EIPx` and `EIP x`, but not `EIP-x`.

The repo [README](https://github.com/ethereum/EIPs) has `EIP-x`, but [the template](https://github.com/ethereum/EIPs/blob/master/eip-template.md) kind of suggests `EIP x` (see `requires` and `replaces` in the header).

Reply:
You are right, this is an unclear topic.

EIP-1 is confusing, it uses "EIP 1" for itself, but "EIPn" for every other reference.

There's an issue discussing (#1464) which was not conclusive.

Lately, the majority of issues and merged EIPs follow the EIP-n notation.

Reply:
https://github.com/ethereum/EIPs/pull/2643